airflow/providers/openlineage/utils/utils.py

```diff
@@ -382,13 +382,6 @@ def get_filtered_unknown_operator_keys(operator: BaseOperator) -> dict:
     return {attr: value for attr, value in operator.__dict__.items() if attr not in not_required_keys}
 
 
-def normalize_sql(sql: str | Iterable[str]):
```

Comment:
If you remove _anything_ from OpenLineage provider, you need to bump minimum dependencies of anything relying on this version. Or, better, don't remove it yet, just remove all usage and mark as deprecated.

Reply:
Makes sense. I marked it as deprecated.